source/standards/principle-least-access.html.md.erb

```diff
@@ -41,6 +41,6 @@ If you’re using the gds-users account to log into your AWS accounts, you shoul
 
 ## Further Guidance
 
-- [NCSC - 10 steps to cyber security](https://www.ncsc.gov.uk/collection/10-steps-to-cyber-security?curPage=/collection/10-steps-to-cyber-security/the-10-steps/managing-user-privileges)
+- [NCSC - 10 steps to cyber security](https://www.ncsc.gov.uk/collection/10-steps)
 - [NCSC - privileged user management](https://www.ncsc.gov.uk/guidance/introduction-identity-and-access-management#section_6)
 - [NIST Special Publication 800-53 -  AC-6 least privilege](https://nvd.nist.gov/800-53/Rev4/control/AC-6)
```

Comment:
This link appears to have the same URL as one above which was change; should it also be updated?

Reply:
Yes! Well spotted.